client/blocks/inline-help/popover.jsx

```diff
@@ -63,6 +66,14 @@ class InlineHelpPopover extends Component {
 		activeSecondaryView: '',
 	};
 
+	componentDidMount() {
+		//If we're in the site preview and eligible for the checklist,
+		//open Checklist Prompt in secondary view
+		if ( this.props.isSitePreview && this.props.isEligibleForDotcomChecklist ) {
```

Comment:
Do we want to do this every time the user is on the site preview? I thought it would only be the first time? Maybe it's better to us a query string to control it? @danhauk what do you think?

Reply:
That's a good question. If we're only showing it for people who came from signup or checkout, we probably could use a query rather than a global call.